backend/src/modules/users_modules/createProfile.js

```diff
@@ -0,0 +1,19 @@
+import { getUsersCollection } from "../db_modules/dbClientConnect.js";
+import bcrypt from "bcryptjs";
+
+export const createProfile = async loginParams => {
+  const collectionUsers = getUsersCollection();
+  if (
+    (await collectionUsers.findOne({ userName: loginParams.userName })) !== null
```

Comment:
Store `await collectionUsers.findOne({ userName: loginParams.userName })` in variable and use it condition